### DISCUSSION TITLE
Tables showing correctly in preview but broken after closing the editor

### DISCUSSION THREAD

**Discussion by mehdi-rashidi** [requarks/wiki#3782] [Help / Questions] [answered]
Hello,
When creating tables in MarkDown Editor, the preview shows the table but after closing the editor, it's rendered as normal text:

**In edit mode:**

![image](https://user-images.githubusercontent.com/78207642/112845795-2f22b100-909d-11eb-8bc7-1708ac8f3f44.png)

**After saving and closing the editor:**

![image](https://user-images.githubusercontent.com/78207642/112845884-482b6200-909d-11eb-802b-a7defd9f5a86.png)
Is it something I'm not doing correctly?

**Comment by adenoz** [comment, marked as answer]
Try this:
```
| Guide Overview ||
|:---: | :---: |
| Complexity | **Senior** |
| Service Affecting | **Yes** |
```
| Guide Overview ||
|:---: | :---: |
| Complexity | **Senior** |
| Service Affecting | **Yes** |

**Comment by SENSEi-James** [comment]
It seems you also need a blank line before and after the table
